### PR TITLE
Adapt mapproxy tests to new autoscale backends

### PR DESCRIPTION
**Pull request by gjn**
This adapts the test urls for mapproxy related tests. It also assures that the wmts getcapabilities contain the correct urls for the re-projected tiles (using the wmts20.[dev|int].bgdi.ch addresses for dev and int and wmts20.geo.admin.ch for prod.

### CODE REVIEW

**Comment by gjn** [comment]
@procrastinatio as we just discussed. Would be great to have this for tomorrow.

File: chsdi/tests/e2e/test_wmtsgettile.py [hunk 1]
```diff
@@ -16,11 +16,11 @@
...
-    'wmts10.geo.admin.ch',
-    'wmts11.geo.admin.ch',
-    'wmts12.geo.admin.ch',
-    'wmts13.geo.admin.ch',
-    'wmts14.geo.admin.ch',
+    'wmts20.geo.admin.ch',
+    'wmts21.geo.admin.ch',
+    'wmts22.geo.admin.ch',
+    'wmts23.geo.admin.ch',
+    'wmts24.geo.admin.ch',
```
Comment (gjn) [inline chsdi/tests/e2e/test_wmtsgettile.py]: I will also add 10-14 again.

**Comment by procrastinatio** [comment]
OK, I thinks it's good, but how do we reconcilie discrepancies between the `GetCapabilities`on `dev` using the BOD and `mapproxy` autoscaling normally `layersConfig`from `prod`. We have to make sure that both are in sync.

**Comment by gjn** [comment]
Yes, that's something we need to take care of. Let's discuss it Friday.

**Pull request merged**